### PR TITLE
abbreviate_artistsort: Get rid of print statements

### PR DESCRIPTION
**Pull request by mineo**
Picards logging module is already imported, there's no need to reinvent debug
levels etc. here

I've loaded a classical album with this plugin active, the plugin showed some warnings, but the release loaded, so I suppose everything still works.

### CODE REVIEW

**Comment by Sophist-UK** [comment]
I have no idea why I used print rather than log when I wrote this!!!

But don't forget to increment the version number.

**Review by zas — APPROVED** [review]

**Comment by zas** [comment]
Those changes should be ported to 2.0 version too

File: plugins/abbreviate_artistsort/abbreviate_artistsort.py [hunk 1]
```diff
@@ -228,13 +217,11 @@ def abbreviate_artistsort(tagger, metadata, track, release):
                         unsorts[i],
                         new_unsort,
                     )
-                    if _debug_level > 0:
-                        print()
-                        print("Error: Unsorted text for %s has changed from '%s' to '%s'!" % (unsortTag, unsorts[i], new_unsort))
-                        print()
+                    log.warning()
+                    log.warning("Error: Unsorted text for %s has changed from '%s' to '%s'!" % (unsortTag, unsorts[i], new_unsort))
+                    log.warning()
```
Comment (Freso) [inline plugins/abbreviate_artistsort/abbreviate_artistsort.py:222]: These empty `log.warning()`s are probably unneeded now, no?

Comment (zas) [reply plugins/abbreviate_artistsort/abbreviate_artistsort.py:222]: > These empty log.warning()s are probably unneeded now, no?

Yes, i missed those, totally unneeded ;)

**Pull request merged**